Cargo.toml

```diff
@@ -36,6 +36,8 @@ serde = ["dep:serde"]
 ndarray-bindings = ["dep:ndarray"]
 datasets = ["dep:rand_distr", "std_rand", "serde"]
 std_rand = ["rand/std_rng", "rand/std"]
+# used by wasm32-unknown-unknown
```

Comment:
Maybe we could add here in the comment that is using by wasm32-unknown-unknown when we want to target browser